### DISCUSSION TITLE
Pin message

### DISCUSSION THREAD

**Discussion by fabston** [tweepy/tweepy#1512] [Questions] [answered]
Hi,

I was wondering if I am able to also Pin a message via the API?

Thanks for your help.

**Comment by Harmon758** [comment, marked as answer]
No, this isn't possible with Twitter's API.
See https://twittercommunity.com/t/pin-a-tweet-with-the-twitter-api/13048 and https://twitterdevfeedback.uservoice.com/forums/930250-twitter-api/suggestions/41253373-pin-unpin-tweets.

**Comment by fabston** [comment]
Ok. Thanks for the quick response.
